pkg/mover/chart.go

```diff
@@ -135,13 +138,13 @@ func (cm *ChartMover) WithRetries(retries uint) *ChartMover {
 // the expected rewrites in the Helm Chart.
 func (cm *ChartMover) Print() {
 	log := cm.logger
-	log.Println("Image moves:")
+	log.Println("Image copies:")
 	for _, change := range cm.imageChanges {
-		pushRequiredTxt := ""
+		pushRequiredTxt := "already exists"
```

Comment:
I find easier to always show the status of the image. I am also thinking about an evolution of this UI potentially in a table based UI.